assets/css/custom/program.scss

```diff
@@ -1,114 +1,84 @@
-@import '../../../public/bootstrap/scss/functions';
-@import '../base/variables';
-@import '../base/mixins';
-@import '../../../public/bootstrap/scss/variables';
-@import '../../../public/bootstrap/scss/mixins';
-@import 'theme_settings';
+@import "../../../public/bootstrap/scss/functions";
+@import "../base/variables";
+@import "../base/mixins";
+@import "../../../public/bootstrap/scss/variables";
+@import "../../../public/bootstrap/scss/mixins";
+@import "theme_settings";
+@import "@material/circular-progress";
 
-button, .button {
-  color: white;
+button,
+.button {
+  color: #fff;
 }
 
 .text-area-empty {
+  border-color: #f00 !important;
   border-width: medium;
-  border-color: red !important;
 }
 
-$thumbnail-size: 300px;
+$thumbnail-size: 200px;
```

Comment:
ticket description was highlighting to get rid of fixed px sizes and use the bootstrap grid.